Client/Frontend/Strings.swift

```diff
@@ -2056,6 +2056,36 @@ extension String {
         value: "Cannot Open Page",
         comment: "Title of the message shown when the user attempts to navigate to an invalid link.",
         lastUpdated: .unknown)
+    public static let CouldntDownloadWallpaperErrorTitle = MZLocalizedString(
+        "Wallpaper.Download.Error.Title.v106",
+        value: "Couldn’t Download Wallpaper",
+        comment: "The title of the error displayed to a user when they try change a wallpaper that failed downloading.",
+        lastUpdated: .v106)
+    public static let CouldntDownloadWallpaperErrorBody = MZLocalizedString(
+        "Wallpaper.Download.Error.Title.v106",
+        value: "Something went wrong with your download.",
+        comment: "The message of the error displayed to a user when they try change a wallpaper that failed downloading.",
+        lastUpdated: .v106)
+    public static let CouldntChangeWallpaperErrorTitle = MZLocalizedString(
+        "Wallpaper.Change.Error.Title.v106",
+        value: "Couldn’t Change Wallpaper",
+        comment: "The title of the error displayed to a user when they trying to change a wallpaper failed.",
+        lastUpdated: .v106)
+    public static let CouldntChangeWallpaperErrorBody = MZLocalizedString(
+        "Wallpaper.Change.Error.Title.v106",
+        value: "Something went wrong with this wallpaper.",
+        comment: "The message of the error displayed to a user when they trying to change a wallpaper failed.",
+        lastUpdated: .v106)
+    public static let WallpaperErrorTryAgain = MZLocalizedString(
+        "Wallpaper.Error.TryAgain.v106",
+        value: "Try Again",
+        comment: "An action for the error displayed to a user when they trying to change a wallpaper failed which allows them to try again.",
```

Comment:
```suggestion
        comment: "Action displayed when changing wallpaper fails.",
```

Same for the next one.